src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,43 @@
+package bit.javaoop;
+
+
+public class CEO extends AbstractEmployee{
+    private final String position="ceo";
+    private String name;
+
+    @Override
+    public String getPosition() {
+        return position;
+    }
+
+    private static CEO ceo;
+    private CEO(){}
```

Comment:
Singleton is kind of an anti-pattern in the OOP world. I don't think it's necessary to implement CEO as a singleton a company can have only one CEO (at a time) but it does not mean that there has to be exactly one instance of CEO in the whole system.